GTViewController/Classes/Nav Classes/GTViewController.m

```diff
@@ -1015,6 +1022,8 @@ - (void)hideNavToolbar {
         
     }
     
+    self.navToolbarIsShown		= NO;
```

Comment:
ok, only put it back if the race condition has been solved. I'm not sure it would matter if the nav bar got prematurely dismissed with this variable being set too early